webpack.config.js

```diff
@@ -5,16 +5,31 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
 const webpack = require('webpack');
 
+let jsLoaders = ['babel-loader'];
+let entry = ['./src/index.js'];
+
 module.exports = (_env, argv = {}) => {
   const isDevMode = argv.mode === 'development';
   const isTestMode = argv.mode === 'test';
 
+  if (isDevMode) {
```

Comment:
Updates to this file mirror the example from https://github.com/noppa/ng-hot-reload/blob/v3.3.1/packages/demo/webpack.config.js#L12